gulp/dist.js

```diff
@@ -1,3 +1,4 @@
+const { version } = require('@ministryofjustice/frontend/package.json')
```

Comment:
Will this be correct? At the time of running this script, the version will be the _current_ version, and then once the script has built, then semantic release will run, which will then update the version in the published package?

I haven't got to a solution for this yet, as its a bit of a pain.